hw_3/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Registration Form</title>
+</head>
+<body>
+    <h1>Registration Form</h1>
+
+    <form method="post">
+        {{ form.hidden_tag() }}
+
+        <label for="first_name">First Name:</label>
+        {{ form.first_name(size=20) }}<br>
+
+        <label for="last_name">Last Name:</label>
+        {{ form.last_name(size=20) }}<br>
+
+        <label for="email">Email:</label>
+        {{ form.email(size=30) }}<br>
+
+        <label for="password">Password:</label>
+        {{ form.password(size=20) }}<br>
+
+        <label for="confirm_password">Confirm Password:</label>
+        {{ form.confirm_password(size=20) }}<br>
+
+        <input type="submit" value="Зарегистрироваться">
```

Comment:
Нужно избавиться от повторений кода
```suggestion
        {{ form.hidden_tag() }}
        {{ form.csrf_token }}
        {% for field in form if field.name != 'csrf_token' %}

          {{ field.label }}: {{ form(size=20) }}<br>
          
          <ul class="alert alert-danger">
	      {% for error in field.errors %}
	      <li>{{ error }}</li>
	      {% endfor %}
	  </ul>
	
        {% endfor %}
        <p>{{ form.submit() }}</p>
```
Причем строки 
```html
        <ul class="alert alert-danger">
	    {% for error in field.errors %}
	    <li>{{ error }}</li>
	    {% endfor %}
	</ul>
```
Повторяют сообщения flash об ошибках. Можно оставить что-то одно. Решать вам, что и где использовать